components/config/src/config/markup.rs

```diff
@@ -36,6 +36,8 @@ pub struct Markdown {
     pub highlight_themes_css: Vec<ThemeCss>,
     /// Whether to render emoji aliases (e.g.: :smile: => 😄) in the markdown files
     pub render_emoji: bool,
+    /// CSS class to add to external links
+    pub external_links_class: Option<String>,
```

Comment:
ah maybe just one thing, should we error if the string contains `"`?

Reply:
Setting `external_links_class = '"external"'` generates:

```html
<a external""="" href="https://www.getzola.org">Zola</a>
```

I could see someone trying to use quotes to assign two classes to external links, so validation there might help.

Since CSS classes can't include quotes (neither `'` nor `"`), should we check for both?

Would you prefer validation be done in `construct_external_link_tag`, when we try to use the class? Or early in `impl Markdown` (`markup.rs`):

```rust
impl Markdown {
    pub fn validate_config(&self) -> Result<()> {
        // Validate external link class doesn't contain quotes which would break HTML and aren't valid in CSS
        if let Some(class) = &self.external_links_class {
            if class.contains('"') || class.contains('\'') {
                bail!(
                    "External link class '{}' contains quotes which are not valid in CSS class names",
                    class
                )
            }
        }
        Ok(())
    }
// …
```

Reply:
> Since CSS classes can't include quotes (neither ' nor "), should we check for both?

Yes. And definitely early in the config for the error checking

Reply:
Done! I did not add a test to validate errors since (1) I didn't find other tests checking for specific error messages and (2) `external_links_class` validation happens in `Config::from_file()`, outside the test infrastructure.